azure-pipelines.yml

```diff
@@ -73,22 +45,6 @@ jobs:
             displayName: "npm pack"
           - script: mv durable-functions-*.tgz package.tgz
             displayName: "Rename package file"
-          - task: CopyFiles@2
-            displayName: "Create smoke test app"
-            inputs:
-                SourceFolder: "$(System.DefaultWorkingDirectory)/test/test-app"
-                Contents: "**"
-                TargetFolder: "$(Agent.BuildDirectory)/test-app"
-                CleanTargetFolder: true
-          - script: npm install $(System.DefaultWorkingDirectory)/package.tgz
-            displayName: "Install packed durable-functions module (test app)"
-            workingDirectory: $(Agent.BuildDirectory)/test-app
-          - script: npm install
-            displayName: "npm install (test app)"
-            workingDirectory: $(Agent.BuildDirectory)/test-app
-          - script: npm run build
-            displayName: "Build smoke test app"
-            workingDirectory: "$(Agent.BuildDirectory)/test-app"

```

Comment:
The smoke test step currently would try to build an old programming model app with this build of the SDK, which would fail. Ideally, I'd like to replace the smoke test step with a new app that follows the new programming model, but it's hard to do this in this PR when the API for the new programming model isn't finalized. I've created an issue to track it though: https://github.com/Azure/azure-functions-durable-js/issues/394